modules/core/src/mathfuncs.cpp

```diff
@@ -2023,4 +2024,382 @@ void cvSolvePoly(const CvMat* a, CvMat *r, int maxiter, int)
+    .63068982562619868570408243613201193511500,    .5322245322245322245322245322245322245322,
+    .63276666957103777644277897707070223987100,    .5311203319502074688796680497925311203320,
+    .63483920917301017716738442686619237065300,    .5300207039337474120082815734989648033126,
+    .63690746223706917739093569252872839570050,    .5289256198347107438016528925619834710744,
+    .63897144645792069983514238629140891134750,    .5278350515463917525773195876288659793814,
+    .64103117942093124081992527862894348800200,    .5267489711934156378600823045267489711934,
+    .64308667860302726193566513757104985415950,    .5256673511293634496919917864476386036961,
+    .64513796137358470073053240412264131009600,    .5245901639344262295081967213114754098361,
+    .64718504499530948859131740391603671014300,    .5235173824130879345603271983640081799591,
+    .64922794662510974195157587018911726772800,    .5224489795918367346938775510204081632653,
+    .65126668331495807251485530287027359008800,    .5213849287169042769857433808553971486762,
+    .65330127201274557080523663898929953575150,    .5203252032520325203252032520325203252033,
+    .65533172956312757406749369692988693714150,    .5192697768762677484787018255578093306288,
+    .65735807270835999727154330685152672231200,    .5182186234817813765182186234817813765182,
+    .65938031808912778153342060249997302889800,    .5171717171717171717171717171717171717172,
+    .66139848224536490484126716182800009846700,    .5161290322580645161290322580645161290323,
+    .66341258161706617713093692145776003599150,    .5150905432595573440643863179074446680080,
+    .66542263254509037562201001492212526500250,    .5140562248995983935742971887550200803213,
+    .66742865127195616370414654738851822912700,    .5130260521042084168336673346693386773547,
+    .66943065394262923906154583164607174694550,    .5120000000000000000000000000000000000000,
+    .67142865660530226534774556057527661323550,    .5109780439121756487025948103792415169661,
+    .67342267521216669923234121597488410770900,    .5099601593625498007968127490039840637450,
+    .67541272562017662384192817626171745359900,    .5089463220675944333996023856858846918489,
+    .67739882359180603188519853574689477682100,    .5079365079365079365079365079365079365079,
+    .67938098479579733801614338517538271844400,    .5069306930693069306930693069306930693069,
+    .68135922480790300781450241629499942064300,    .5059288537549407114624505928853754940711,
+    .68333355911162063645036823800182901322850,    .5049309664694280078895463510848126232742,
+    .68530400309891936760919861626462079584600,    .5039370078740157480314960629921259842520,
+    .68727057207096020619019327568821609020250,    .5029469548133595284872298624754420432220,
+    .68923328123880889251040571252815425395950,    .5019607843137254901960784313725490196078,
+    .69314718055994530941723212145818, 5.0e-01,
+};
+
+const double* getLogTab()
```

Comment:
could you please rename it to `getLogTab64f()` and `getLogTab32f()`, respectively, in order to be more consistent with HAL notation?